api/v1alpha1/generated.proto

```diff
@@ -658,6 +672,24 @@ message LocalObjectReference {
   optional string name = 3;
 }
 
+// RetryPolicy describes a policy for retrying a connection until the
+// connection is either refused or times out.
```

Comment:
Envoy has more fine-grained [policy setting](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/router_filter#x-envoy-retry-on).

And Knative (net-contour) leverages it - https://github.com/knative-sandbox/net-contour/blob/42ebe377c7c9d1e62fe4265ff3ef0cee285e8be3/pkg/reconciler/contour/resources/httpproxy.go#L80-L95
So it would be great if we could have some optional filed to customize it.

Reply:
Also, we would like to know the default retry policy.